tbx/services/models.py

```diff
@@ -172,6 +172,10 @@ def get_featured_case_studies(self):
             if f.case_study
         ]
 
+    @property
+    def case_studies_filter_by(self):
+        return self.slug
+
```

Comment:
Ah, I missed it in my earlier code review that instead of slug, you could use the `service` related field, which would give better results as it links directly to the service taxonomy and will match the filter on the WorkIndexPage. 

Also since the subservicepage has a `service` property to mirror the related field, you will only need to define this at `BaseServicePage`.

E.g.

```suggestion
    @property
    def case_studies_filter_by(self):
        if self.service:
            return self.service.slug
        
        # If no service defined, don't filter by anything
        return ""
```